scss/styles.scss

```diff
@@ -1,10 +1,24 @@
+// variables
+$breackpoint: 1023px;
```

Comment:
This variable is likely to be used many times across many files instead of defining this variable 2 times you can define it for example in the application.scss or create and import another .scss file. Keep this in mind when you create variables :wink: